app/cli/pkg/action/attestation_init.go

```diff
@@ -219,6 +229,19 @@ func (action *AttestationInit) Run(ctx context.Context, opts *AttestationInitRun
 		attestationID = workflowRun.GetId()
 	}
 
+	// Get CAS credentials for PR metadata upload
+	var casBackend = &casclient.CASBackend{Name: "not-set"}
+	if !action.dryRun && attestationID != "" {
+		connectionCloserFn, err := getCASBackend(ctx, client, attestationID, action.casCAPath, action.casURI, action.connectionInsecure, action.Logger, casBackend)
+		if err != nil {
+			// This should never happen in lenient mode, but handle it just in case
```

Comment:
in `att add` the error is being returned, why is it different in this case? (lenient mode?)

Reply:
It's a best effort in the `attestation init`. I called it `lenient mode` that basically means, don't fail. It was from a previous attempt in the implementation. I've just removed the comment and replace it for one that indicates that this is a best effort for PR/MR metadata.